infrastructure/router/router.go

```
@@ -0,0 +1,18 @@
+package router
+
+import (
+	"academy_bootcamp/interface/controller"
+
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/middleware"
+)
+
+func NewRouter(e *echo.Echo, c controller.AppController) *echo.Echo {
+	e.Use(middleware.Logger())
+	e.Use(middleware.Recover())
+
+	e.GET("/items/:id", func(context echo.Context) error { return c.Item.GetItemById(context) })
```

You should never ignore errors; remember to also wrap and propper handle it.